### PR TITLE
Added Testing strategy section - first cut

### PR DESCRIPTION
**Pull request by kauerrolemodel**

### CODE REVIEW

File: support/project_readme.md [hunk 1]
```diff
@@ -76,6 +76,24 @@ bundle install
...
+
+### Other tests
+At this point, no other tests are being employed.
+However, one might consider performance tests or other categories and describe the reasons here
+
+## Testing tools
+Which tools are we using?
+
+## Continuous integration
+No CI has been set up yet (though we recommend Sempahore in most cases).  When you set it up, tell what you need to know here
```
Comment (calebwoods) [inline support/project_readme.md]: CI is usually linked elsewhere in the document.  Not sure we need to repeat it here

File: support/project_readme.md [hunk 2]
```diff
@@ -76,6 +76,24 @@ bundle install
 * [App Status Page](http://app.<applicationname>.com/_ping) will give you information about what is running.
 * Alternatively, you can ssh in and check that the application server and web server are both running.
 
+# Testing Strategy
+## Testing approach
+### End User tests
+Due to the nature of this application, End User tests are...
```
Comment (calebwoods) [inline support/project_readme.md]: Should this include a line like: "To run the End User tests: bundle exec spinach"

File: support/project_readme.md (hunk 1 above)
Comment (wesrich) [reply support/project_readme.md]: @calebwoods: The only place I see (in this document) is in the "Links to" section below, with possibly a mention of it under deployment processes (if it's set up to do CI/CD).

I think it's probably fine to put specifics here though. For example, if there's a recurring nightly build of the Staging branch, or a specific branch that builds on Browserstack every night, or some other oddity ... it's probably worth having this section (and just taking it out if it doesn't apply). 😃

Comment (calebwoods) [reply support/project_readme.md]: That seems reasonable

File: support/project_readme.md (hunk 2 above)
Comment (calebwoods) [reply support/project_readme.md]: This seems covered by the other section

**Pull request merged**